[2022-04-13 22:36:14,921] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: LocalIngestionDAG.ingest scheduled__2022-04-02T06:00:32+00:00 [queued]>
[2022-04-13 22:36:14,948] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: LocalIngestionDAG.ingest scheduled__2022-04-02T06:00:32+00:00 [queued]>
[2022-04-13 22:36:14,949] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-04-13 22:36:14,950] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-04-13 22:36:14,951] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-04-13 22:36:14,991] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): ingest> on 2022-04-02 06:00:32+00:00
[2022-04-13 22:36:15,000] {standard_task_runner.py:52} INFO - Started process 11336 to run task
[2022-04-13 22:36:15,012] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'LocalIngestionDAG', 'ingest', 'scheduled__2022-04-02T06:00:32+00:00', '--job-id', '1958', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_local.py', '--cfg-path', '/tmp/tmpoiht09f8', '--error-file', '/tmp/tmpc6bjj_1j']
[2022-04-13 22:36:15,016] {standard_task_runner.py:77} INFO - Job 1958: Subtask ingest
[2022-04-13 22:36:15,171] {logging_mixin.py:109} INFO - Running <TaskInstance: LocalIngestionDAG.ingest scheduled__2022-04-02T06:00:32+00:00 [running]> on host 13db72185839
[2022-04-13 22:36:15,301] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=LocalIngestionDAG
AIRFLOW_CTX_TASK_ID=ingest
AIRFLOW_CTX_EXECUTION_DATE=2022-04-02T06:00:32+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-02T06:00:32+00:00
[2022-04-13 22:36:15,303] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-04-13 22:36:15,305] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'ls /opt/***']
[2022-04-13 22:36:15,331] {subprocess.py:85} INFO - Output:
[2022-04-13 22:36:15,344] {subprocess.py:89} INFO - ***.cfg
[2022-04-13 22:36:15,345] {subprocess.py:89} INFO - dags
[2022-04-13 22:36:15,346] {subprocess.py:89} INFO - logs
[2022-04-13 22:36:15,347] {subprocess.py:89} INFO - output.csv
[2022-04-13 22:36:15,348] {subprocess.py:89} INFO - output_2022-03.csv
[2022-04-13 22:36:15,349] {subprocess.py:89} INFO - output_2022-04.csv
[2022-04-13 22:36:15,350] {subprocess.py:89} INFO - plugins
[2022-04-13 22:36:15,350] {subprocess.py:89} INFO - requirements.txt
[2022-04-13 22:36:15,351] {subprocess.py:89} INFO - scripts
[2022-04-13 22:36:15,352] {subprocess.py:89} INFO - webserver_config.py
[2022-04-13 22:36:15,352] {subprocess.py:93} INFO - Command exited with return code 0
[2022-04-13 22:36:15,425] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=LocalIngestionDAG, task_id=ingest, execution_date=20220402T060032, start_date=20220413T223614, end_date=20220413T223615
[2022-04-13 22:36:15,465] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-13 22:36:15,537] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
